Apps/Admin/Server/Services/SupportService.cs

```diff
@@ -217,19 +217,19 @@ private async Task<IEnumerable<PatientSupportDependentInfo>> GetAllDependentInfo
 
         private async Task<VaccineDetails> GetVaccineDetailsAsync(PatientModel patient, bool refresh, CancellationToken ct)
         {
-            if (!string.IsNullOrEmpty(patient.Phn) && patient.Birthdate != DateTime.MinValue)
+            if (string.IsNullOrEmpty(patient.Phn) || patient.Birthdate == DateTime.MinValue)
             {
-                return await immunizationAdminDelegate.GetVaccineDetailsWithRetriesAsync(patient.Phn, await this.GetAccessTokenAsync(ct), refresh, ct);
+                logger.LogError("Patient PHN {PersonalHealthNumber} or DOB {Birthdate}) are invalid", patient.Phn, patient.Birthdate);
+                throw new InvalidDataException(ErrorMessages.PhnOrDateAndBirthInvalid);
```

Comment:
Just noticed this error message is labelled "DateAndBirth" instead of "DateOfBirth".